imports/ui/components/SearchBar.jsx

```diff
@@ -0,0 +1,294 @@
+import React from 'react';
+import clsx from 'clsx';
+import Select from 'react-select';
+import { emphasize, makeStyles, useTheme } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import NoSsr from '@material-ui/core/NoSsr';
+import TextField from '@material-ui/core/TextField';
+import Paper from '@material-ui/core/Paper';
+import Chip from '@material-ui/core/Chip';
+import MenuItem from '@material-ui/core/MenuItem';
+import CancelIcon from '@material-ui/icons/Cancel';
+import PropTypes from 'prop-types';
+
+const suggestions = [
+  { label: 'Apples' },
+  { label: 'Bananas' },
+  { label: 'Oranges' }
+].map(suggestion => ({
+  value: suggestion.label,
+  label: suggestion.label,
+}));
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    flexGrow: 1,
+    height: 'auto',
+    backgroundColor: 'white',
+    padding: 10,
+    alignItems: 'center',
```

Comment:
Can you apply prettier with double quotes and no commas on last lines on the whole file?

Reply:
Fixed